packages/compiler/src/server/compile-service.ts

```diff
@@ -351,6 +351,27 @@ export function createCompileService({
         await fileSystemCache.setData(pkgPath, pkg ?? {});
       }
 
+      const entrypoints = clientConfigsProvider?.config?.compile?.entrypoint;
+      if (entrypoints && entrypoints.length > 0) {
```

Comment:
we should honor this settings if user set it which means this setting has a higher priority than the one defined in package.json. same comment for below. thx